static/js/stream_create.js

```diff
@@ -150,20 +150,17 @@ function create_stream() {
     }
     data.subscriptions = JSON.stringify([{name: stream_name, description}]);
 
-    let invite_only;
-    let history_public_to_subscribers;
     const privacy_setting = $("#stream_creation_form input[name=privacy]:checked").val();
 
-    if (privacy_setting === "invite-only") {
-        invite_only = true;
-        history_public_to_subscribers = false;
-    } else if (privacy_setting === "invite-only-public-history") {
-        invite_only = true;
-        history_public_to_subscribers = true;
-    } else {
-        invite_only = false;
-        history_public_to_subscribers = true;
-    }
+    const is_web_public = ["is-web-public"].includes(privacy_setting);
+    const invite_only = ["invite-only", "invite-only-public-history"].includes(privacy_setting);
+    const history_public_to_subscribers = [
+        "invite-only-public-history",
+        "is-web-public",
+        "public",
+    ].includes(privacy_setting);
```

Comment:
I think the rewrite to use `includes` is probably less readable than the old logic; I'd change this to just do it the verbose way.